backend/cmd/headlamp.go

```diff
@@ -209,14 +210,27 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 
 	// KubeConfig clusters
 	if kubeConfigPath != "" {
-		var err error
-
-		contextsFound, err := GetContextsFromKubeConfigFile(kubeConfigPath)
-		if err != nil {
-			log.Println("Failed to get contexts from", kubeConfigPath, err)
+		var kubeConfigs []string
+		if runtime.GOOS == "windows" {
+			kubeConfigs = strings.Split(kubeConfigPath, ";")
+		} else {
+			kubeConfigs = strings.Split(kubeConfigPath, ":")
```

Comment:
Nit: I'd assign the right delimiter to a var and avoid duplicating the logic (i.e. calling just one Split).